### PR TITLE
Add BookSuggestion model and controller

### PR DESCRIPTION
**Pull request by ziflit**
## Summary
Adds a BookSuggestion model, with a controller that exposes `#create` and tests to validate
both.
## Trello card(s)
https://trello.com/c/iZeymf0m (model)
https://trello.com/c/8yjxmfVU (controller)
https://trello.com/c/YlDDkWu0 (tests)

### CODE REVIEW

File: spec/controllers/api/v1/book_suggestion_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+          before do
+            post :create, params: { title: suggestion.title, author: suggestion.author,
+                                    publisher: suggestion.publisher, year: suggestion.year,
+                                    link: suggestion.link, editorial: suggestion.editorial,
+                                    price: suggestion.price, user_id: user }
+          end
+          it 'creates a new rent' do
+            expect do
+              post :create, params: { title: suggestion.title, author: suggestion.author,
```
Comment (ziflit) [inline spec/controllers/api/v1/book_suggestion_controller_spec.rb]: Couldn't find a way to check that `BookSuggestion.count` increments by one without explicitely writing the `post` twice.

File: app/controllers/api/v1/book_suggestion_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
+      skip_before_action :authenticate_request, only: [:create]
+      def create
+        book_suggestion = BookSuggestion.new(params.permit(
+                                               :title, :author, :publisher, :year, :link,
+                                               :editorial, :user_id, :price
+        ))
+        if book_suggestion.save
```
Comment (alebian) [inline app/controllers/api/v1/book_suggestion_controller.rb]: return head :created if book_suggestion.save

File: config/routes.rb [hunk 3]
```diff
@@ -14,5 +14,6 @@
     end
 
     resources :books, only: [:show, :index]
+    resources :book_suggestion, only: [:create]
```
Comment (alebian) [inline config/routes.rb]: book_suggestions here

Remember that in REST, resources should be plural

File: spec/controllers/api/v1/book_suggestion_controller_spec.rb (hunk 1 above)
Comment (alebian) [inline spec/controllers/api/v1/book_suggestion_controller_spec.rb]: It's fine like this. Anyway, here you are making 2 requests, one in the before and one in the create. Remove the before block, and add the request manually in the test below

**Pull request merged**